scripts/packages/ui/src/components/notifications/__tests__/ToastContainer.test.tsx

```
@@ -0,0 +1,375 @@
+// Mock the Toast component
+jest.mock('../Toast', () => ({
+  Toast: ({
+    notification,
+    onDismiss,
+  }: {
+    notification: Notification;
+    onDismiss: (id: string) => void;
+  }) => (
+    <div data-testid={`toast-${notification.id}`}>
+      <span>{notification.title}</span>
+      <button onClick={() => onDismiss(notification.id)}>Dismiss</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)